e2e/tests/network_create.go

```diff
@@ -172,6 +184,55 @@ func NetworkCreate(opt *option.Option) {
 				Expect(httpResponse).Should(HaveHTTPStatus(http.StatusNotFound))
 			})
 		})
+
+		When("a network create request is made with network option com.docker.network.bridge.enable_icc set to false", func() {
+			It("should return 201 Created and the network ID", func() {
+				testBridge := "br-test"
+				request := types.NewCreateNetworkRequest(testNetwork, withEnableICCdNetworkOptions("false", testBridge)...)
+
+				httpResponse := createNetwork(*request)
+				Expect(httpResponse).Should(HaveHTTPStatus(http.StatusCreated))
+
+				response := unmarshallHTTPResponse(httpResponse)
+				Expect(response.ID).ShouldNot(BeEmpty())
+				Expect(response.Warning).Should(BeEmpty())
+
+				DeferCleanup(cleanupNetworkWithHTTP(testNetwork))
+
+				stdout := command.Stdout(opt, "network", "inspect", testNetwork)
+				Expect(stdout).To(ContainSubstring(`"finch.network.bridge.enable_icc.ipv4": "false"`))
+
+				// check iptables rules exists
+				iptOpt, _ := option.New([]string{"iptables"})
+				command.Run(iptOpt, "-C", "FINCH-ISOLATE-CHAIN",
+					"-i", testBridge, "-o", testBridge, "-j", "DROP")
+				time.Sleep(30 * time.Second)
```

Comment:
Sync'd offline: to be removed.

Reply:
Good catch. leftover from an earlier local test.